src/github.ts

```diff
@@ -190,4 +195,84 @@ export class GitHub {
       url: artifactURL
     };
   }
+
+  public static async writeBuildSummary(opts: BuildSummaryOpts): Promise<void> {
+    // can't use original core.summary.addLink due to the need to make
+    // EOL optional
+    const addLink = function (text: string, url: string, addEOL = false): string {
+      return `<a href="${url}">${text}</a>` + (addEOL ? os.EOL : '');
+    };
+
+    const refsSize = Object.keys(opts.exportRes.refs).length;
+
+    // prettier-ignore
+    const sum = core.summary
+      .addHeading('Docker Build summary', 1)
+      .addRaw(`<p>`)
+        .addRaw(`For a detailed look at the build, download the following <code>.dockerbuild</code> file and import it into Docker Desktop's Builds view. `)
+        .addBreak()
+        .addRaw(`The build file provides insights into your build, such as timing, dependencies, results, logs, traces, and more. `)
```

Comment:
question: (I might be totally out of line here / missing some context) Do we need to emphasize the `.dockerbuild` extension at all? It seems like the downloaded file is a `.zip` archive image layout. Could we maybe just call it a "build record"? Something like:

```suggestion
        .addRaw(`This is a summary of the build job. For the full details about your build, download the build record and import it into Docker Desktop's Builds view.`)
        .addBreak()
        .addRaw(`The build record includes data such as timing, dependencies, results, logs, traces, and more.`)
```

Reply:
There can be multiple build records within a `.dockerbuild` file (with bake): 

![image](https://github.com/docker/actions-toolkit/assets/1951866/ee8c8eb1-2437-4f38-938e-3eb3c7597cea)

`.dockerbuild` is more like a bundle having the build record and associated blob suc has traces, provenance, ...

Reply:
Maybe we can call the downloadable artifact a "build record archive" then, and avoid calling it `.dockerbuild` file (since that is not very descriptive). A build record archive contains one or more build records, each of which contains the logs, traces, provenance, etc for a single build invocation

Reply:
@dvdksn Updated: https://github.com/docker/actions-toolkit/actions/runs/9094081924?pr=324#summary-24994391676

![image](https://github.com/docker/actions-toolkit/assets/1951866/80420f5c-c378-4607-a3e8-a25c67a1cf28)